static/js/Misc.jsx

```diff
@@ -136,14 +179,21 @@ const ContentText = ({text, html, overrideLanguage, defaultToInterfaceOnBilingua
       return lang === langShort;
     });
   }
-  return renderedItems.map( x =>
-      isDangerouslySetInnerHTML ?
-          <span className={`contentSpan ${x[0]}`} lang={x[0]} key={x[0]} dangerouslySetInnerHTML={{__html: x[1]}}/>
-          :
-          <span className={`contentSpan ${x[0]}`} lang={x[0]} key={x[0]}>{x[1]}</span>
-  );
+  return renderedItems;
+}
+
+const ContentText = (props) => {
+  /* Renders content language throughout the site (content that comes from the database and is not interface language) */
+  const renderedItems = ContentTextFilterByLang({...props});
+  return renderedItems.map(renderItem => <ContentSpan item={renderItem} isDangerouslySetInnerHTML={!!props.html}/>);
 };
 
+const ContentSpan = ({item, isDangerouslySetInnerHTML}) => {
```

Comment:
I would break the prop `item` into two props: One for language and one for content.

Reply:
Can you add a comment explaining this component?

Reply:
Also, I understand the name of `isDangerouslySetInnerHTML` was in the old code, but it seems the prop could just be called `isHTML`, right? Maybe I'm missing something.